### PR TITLE
AAP-2854 added prerequisites for controller and hub

### PR DESCRIPTION
**Pull request by rogrange**
Added prerequisites for controller and hub to section 4.1 prereqs.

### CODE REVIEW

**Comment by lmalivert** [comment]
Hi @rogrange. Do you mind providing a link to the Jira issue that this PR is associated w/so that I can get a better context of this PR?

**Comment by rogrange** [comment]
> 

Here you go Lydie: https://issues.redhat.com/browse/AAP-2854

File: downstream/assemblies/platform/assembly-installing-controller-operator.adoc [hunk 1]
```diff
@@ -28,6 +28,8 @@ When an instance of {ControllerName} is removed, the associated PVCs are not aut
 == Prerequisites
 
 * You have installed the {PlatformName} catalog in Operator Hub.
+* For Controller, a default 'StorageClass' must be configured on the cluster for the operator to dynamically create needed PVC's. This is not necessary if an external PostgreSQL database is configured.
```
Comment (lmalivert) [inline downstream/assemblies/platform/assembly-installing-controller-operator.adoc:31]:
```suggestion
* For Controller, a default StorageClass must be configured on the cluster for the operator to dynamically create needed PVCs. This is not necessary if an external PostgreSQL database is configured.
```

Suggestion to remove apostrophe from `PVCs` `and StorageClass`.  Took a look at other RH guides and `StorageClass` does not have apostrophes. If this was directed by your SME, please disregard my suggestion.

Comment (rogrange) [reply downstream/assemblies/platform/assembly-installing-controller-operator.adoc]: @lmalivert I made the corrections. I am still relying internally upon what i used to do. I appreciate the review!

File: downstream/assemblies/platform/assembly-installing-controller-operator.adoc [hunk 2]
```diff
@@ -28,6 +28,8 @@ When an instance of {ControllerName} is removed, the associated PVCs are not aut
...
+* For Hub a 'StorageClass' that supports 'ReadWriteMany' must be available on the cluster to dynamically created the PVC needed for the content, redis and api pods. If it is not the default 'StorageClass' on the cluster, you can specify it when creating your 'AutomationHub' object.
```
Comment (lmalivert) [inline downstream/assemblies/platform/assembly-installing-controller-operator.adoc]:
```suggestion
* For Hub a StorageClass that supports ReadWriteMany must be available on the cluster to dynamically created the PVC needed for the content, redis and api pods. If it is not the default StorageClass on the cluster, you can specify it when creating your 'AutomationHub' object.
```
Same comment as above regarding `ReadWriteMany` and `StorageClass`.

File: downstream/assemblies/platform/assembly-installing-controller-operator.adoc (hunk 1 above)
Comment (lmalivert) [reply downstream/assemblies/platform/assembly-installing-controller-operator.adoc]: You're welcome!

File: downstream/assemblies/platform/assembly-installing-controller-operator.adoc (hunk 2 above)
Comment (rogrange) [reply downstream/assemblies/platform/assembly-installing-controller-operator.adoc]: So, do those, as they are functions not get any markup at all?

Comment (rogrange) [reply downstream/assemblies/platform/assembly-installing-controller-operator.adoc]: @lmalivert I removed them all. I am not used to not calling out functions, that seems odd to me. :-)

Comment (lmalivert) [reply downstream/assemblies/platform/assembly-installing-controller-operator.adoc]: I took a look at some of the OpenShift docs and neither one of those functions have markup. 
Example for [ReadWriteMany](https://access.redhat.com/documentation/en-us/red_hat_openshift_container_storage/4.8/html/deploying_and_managing_openshift_container_storage_using_google_cloud/managing-persistent-volume-claims_gcp)

Example for [StorageClass](https://access.redhat.com/documentation/en-us/red_hat_openshift_data_foundation/4.9/html/managing_and_allocating_storage_resources/storage-classes_rhodf) 

You are the author, so if you disagree, or was advised differently by an SME, disregard my suggestions.

**Pull request merged**